_posts/2016/2016-05-25-spring-io.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Spring I/O 2016"
+date: "2016-05-25 09:00:00"
+image: blog-header/springio.jpg
+categories: events
+authors: ["Xavi"]
+---
+Last 19th and 20th of May a conference called *Spring I/O* took place in Barcelona. It was the 5th edition of an event whose goal is to spread the word about the well-known framework [Spring](https://spring.io/). Luckily, four *epagees* including myself had the chance to attend the conference.
+
+The conference itself consisted of a number of talks and workshops, up to four at the same time. Each day began with a keynote and, as it was bound to be, there was plenty of coffee available, although considering there were lots of Java developers I think tea would have fit like a glove.
+
+As you might expect, I didn't find all activities equally interesting or fun, so I'll just talk about those I enjoyed the most in no particular order.
+
+# "Cloud Native Java", by Josh Long
```

Comment:
# "Cloud Native Java", by Josh Long > ## "Cloud Native Java", by Josh Long

and add blank line after headline